README.rst

```diff
@@ -125,16 +125,17 @@ You can easily create your own Setting classes for more complex settings.
 .. code:: python
 
     import re
+
     import appsettings
+    from django.core.exceptions import ValidationError
 
 
     class RegexSetting(appsettings.Setting):
-        def checker(self, name, value):
+        def validate(self, value):
             re_type = type(re.compile(r'^$'))
             if not isinstance(value, (re_type, str)):
-                # raise whatever exception
-                raise ValueError('%s must be a a string or a compiled regex '
-                                 '(use re.compile)' % name)
+                # Raise ValidationError
+                raise ValidationError('%s must be a string or a compiled regex (use re.compile)')
```

Comment:
So, will Django populate the `%s` itself?

Reply:
Yes, moreover it's recommended way to raise a `ValidationError`, see https://docs.djangoproject.com/en/2.1/ref/forms/validation/#raising-validationerror

Reply:
Can you simply copy-paste it then :slightly_smiling_face: ? So both examples use the recommended way.

Reply:
Now I get the problem. Thanks :)